test/e2e/features/chat/sending_status/attachments.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Chat attachments have correct sending status
+
+  Scenario: File attachment status changes from `sending` to `sent`
+    Given I am Alice
+    And user Bob
+    And Bob has dialog with me
+    And I am in chat with Bob
+    And I wait until `MessageField` is present
+
+    Given I have Internet with delay of 3 seconds
+    Then I attach "test.txt" file
+    And I wait until `Send` is present
+    Then I tap `Send` widget
+    And I wait until status of "test.txt" attachment is sending
+    And I wait until status of "test.txt" attachment is sent
```

Comment:
@krida2000 @SleepySquash I'd like these steps to be re-done in `Given` -> `When`-> `Then` manner. So, there are explicit "preconditions" -> "actions" -> "expectations".

Reading it while there are only "preconditions" -> "expectations" is kinda weird, especially when expectations do look like some preconditions.